catalog-info.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: 'i18n-tools'
+  description: "Tools to help with internationalization and localization of Open edX projects"
+  links:
+    - url: "https://github.com/openedx/i18n-tools"
+      title: "Source Code"
+      icon: "Code"
+  annotations:
+    github.com/project-slug: openedx/i18n-tools
```

Comment:
You don't need this, see the example here for the latest best practice for catalog-info.yaml: https://docs.openedx.org/projects/openedx-proposals/en/latest/processes/oep-0055/decisions/0001-use-backstage-to-support-maintainers.html#references